domain/service/impl/invitation_service.go

```diff
@@ -10,47 +10,71 @@ import (
 
 type InvitationService struct {
 	invitationStorage IInvitationStorage
-	userStorage       IUserStorage
+	groupStorage      IGroupStorage
 }
 
-func NewInvitationService(invitationStorage *IInvitationStorage, userStorage *IUserStorage) IInvitationService {
-	return &InvitationService{invitationStorage: *invitationStorage, userStorage: *userStorage}
+func NewInvitationService(invitationStorage *IInvitationStorage, groupStorage *IGroupStorage) IInvitationService {
+	return &InvitationService{invitationStorage: *invitationStorage, groupStorage: *groupStorage}
 }
 
-func (i InvitationService) CreateGroupInvitation(request GroupInputDTO, groupID UUID) error {
+func (i InvitationService) CreateGroupInvitation(request GroupInvitationDTO) error {
 	// get invites from request
 	invites := request.Invites
-	// TODO: change, wrong implementation, look up how to store association in gorm
 
-	// handle group invitations
+	// handle group invitations for all invitees
 	for _, invitee := range invites {
-		groupInvitation := model.CreateGroupInvitation(groupID, invitee)
+		groupInvitation := model.CreateGroupInvitation(request.GroupID, invitee)
```

Comment:
How about we keep things consistent and not directly use `model.CreateGroupInvitation` here? Instead, let's create a method in `dto/invitation.go` called `ToGroupInvitationModel` and call `CreateGroupInvitation` from there, just like we do with groups in groups_service.